docs/configurations.md

```diff
@@ -0,0 +1,266 @@
+| scalar.db.dynamo.namespace.prefix         | Prefix for the user namespaces and metadata namespace names. Since AWS requires to have unique tables names in a single AWS region, this is useful if you want to use multiple ScalarDB environments (development, production, etc.) in a single AWS region. |          |
+
+- For JDBC databases, the following configurations are available:
+
+| name                                                    | description                                                                                                                                                                  | default                      |
+|---------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|
+| scalar.db.storage                                       | `jdbc` should be specified.                                                                                                                                                  | -                            |
+| scalar.db.contact_points                                | JDBC connection URL.                                                                                                                                                         |                              |
+| scalar.db.username                                      | Username to access the database.                                                                                                                                             |                              |
+| scalar.db.password                                      | Password to access the database.                                                                                                                                             |                              |
+| scalar.db.jdbc.connection_pool.min_idle                 | Minimum number of idle connections in the connection pool.                                                                                                                   | 20                           |
+| scalar.db.jdbc.connection_pool.max_idle                 | Maximum number of connections that can remain idle in the connection pool.                                                                                                   | 50                           |
+| scalar.db.jdbc.connection_pool.max_total                | maximum total number of idle and borrowed connections that can be active at the same time for the connection pool. Use a negative value for no limit.                        | 100                          |
```

Comment:
```suggestion
| scalar.db.jdbc.connection_pool.max_total                | Maximum total number of idle and borrowed connections that can be active at the same time for the connection pool. Use a negative value for no limit.                        | 100                          |
```
This is a very minor thing. Update lowercase to uppercase at the first character of the sentence.